images/origin/scripts/recycler.sh

```diff
@@ -22,8 +21,20 @@ if [[ ! -d ${dir} ]]; then
     exit 1
 fi
 
+# remount to nfsv3 so not restricted by nfs v4 acls, see nfs4_acl(5)
```

Comment:
there is a [suggestion](https://github.com/openshift/origin/issues/5739#issuecomment-155099730) for doc